misc/netcdf/python/processors/netcdfimagesource.py

```diff
@@ -34,13 +25,8 @@ def processorInfo():
     def getProcessorInfo(self):
         return NetCDFImageSource.processorInfo()
 
-    def reloadData(self):
-        extents = []
-        GenericNetCDFSource.reloadData(self, extents)
-
-        buffer = numpy.concatenate(self.data, axis=2)
-        minVal = numpy.amin(buffer)
-        maxVal = numpy.amax(buffer)
+    def dataLoaded(self, data, extents):
+        buffer = numpy.concatenate(data, axis=2)
```

Comment:
Is the concatenation used to put things in contiguous memory? Could add a comment to explain

Reply:
you have to ask @AnkeAnke